core/load-git.cpp

```diff
@@ -357,22 +357,28 @@ static char *parse_keyvalue_entry(void (*fn)(void *, const char *, const std::st
 		line++;
 	}
 
-	if (c == '=')
+	if (c != 0)
```

Comment:
PS: Could it be that this is actually the only line needed to fix the problem?

Reply:
Answering to myself: no, because the next key would be interpreted as a value. :(